### DISCUSSION TITLE
Stop ripgrep output from shouting at me

### DISCUSSION THREAD

**Discussion by tim-gq** [qltysh/qlty#1372] [Feature Requests]
### Problem statement

When using ripgrep to find todo's etc, the comment that it leaves on my pull requests copies the comment.  If it's a comment from ruby, it starts with a `# `.  This is causing Github to markdown parse the comment, and insert the comment into the pull request in header-size text.  Is very oppressive.  :-P

Example:

# TODO: do this thing

### Solution brainstorm (optional)

Wrap the comment found like this: "Ripgrep detection: `# TODO: do this thing`" or something similar

**Comment by brynary** [comment]
Hi @tim-gq -- This feature was implemented by @marschattha and deployed to production!

**Reply by tim-gq** [reply]
Awesome, thanks so much @marschattha.  🎉 

And thank you to the Qlty team for prioritizing this.
